test/rules/order-imports.js

```diff
@@ -1471,5 +1471,31 @@ comment3 */", // the spacing here is really sensitive
 				},
 			],
 		}),
+		// Option unassignedImports: 'allow' should consider unassigned imports when sorting for respective groups
+		test({
+			code: `
+				import './an-unassigned-relative';
+				import path from 'path';
+				import _ from './relative';
+				import 'an-unassigned-module';
+		      `,
+			output: `
+				import './an-unassigned-relative';
+				import path from 'path';
+				import _ from './relative';
+				import 'an-unassigned-module';
+		      `,
```

Comment:
Does `eslint` compare this output after running '--fix' against this plugin. If so, shouldn't this be re-ordered because when running locally against a file, the fixer does re-order as per the errors mentioned below.

Probably I'm missing something here?